Modules/posixmodule.c

```diff
@@ -2634,11 +2634,14 @@ _posix_free(void *module)
 static PyObject *
 stat_nanosecond_timestamp(_posixstate *state, time_t sec, unsigned long nsec)
 {
+#if SIZEOF_LONG >= 8
```

Comment:
I do not think this is correct. The comparison is always true, not always false if SIZEOF_LONG < 8.